docker/constraints.txt

```
@@ -55,6 +55,7 @@ google-resumable-media==1.2.0
 googleapis-common-protos==1.53.0
 greenlet==1.1.0
 HeapDict==1.0.1
+hdstats==0.2.1
```

I think this should be 
```suggestion
hdstats==hdstats-0.1.8.post1
```

https://packages.dea.ga.gov.au/hdstats/